docs/workflows.rst

```diff
@@ -0,0 +1,13 @@
+=========
+Workflows 
+=========
```

Comment:
Should we consider moving this into the workflows package?
:thinking:

Reply:
I've called the section "Workflows" because it largely deals with things that are just about `python-workflows`  but some of the stuff is more about how Zocalo as a whole operates. We could put some of it, or a different version of it in the workflows package. Or change the section name slightly?